src/assets/globals.scss

```diff
@@ -73,19 +73,16 @@ body {
 
 .scroll {
     overflow: auto;
-    scrollbar-color: transparent transparent; /* For Firefox */
 }
 
 .scroll::-webkit-scrollbar-thumb {
-    background-color: transparent;
-}
-
-.scroll:hover {
-    scrollbar-color: var(--dl-color-lighter);
+    background-color: var(--dl-color-lighter);
```

Comment:
@makc-dataloop this will cause it so its always visible. we would like to have it always transparent and customize it for either firefox / chromes needs. 
Lets discuss this further tomorrow.